app.py

```diff
@@ -1,37 +1,38 @@
 from flask import Flask, render_template, request, session, redirect, url_for
 import view_model as view_model
-import db_items 
+from db_items import ItemsDb
 
-def create_app():
-    app = Flask(__name__)    
+itemDb = ItemsDb()
 
+def create_app(itemDb):
```

Comment:
This change will break the prod container, because it expects to get an app through `app:create_app()` without specifying any arguments. Since you create a default app below, we can fix that by adjusting your `run.sh` file to just use that, i.e. `"app:app"` instead of `"app:create_app()"`